docs/root/intro/version_history.rst

```diff
@@ -4,6 +4,7 @@ Version history
 1.7.0
 =====
 
+* listeners: added :ref:`tcp_fast_open_queue_length <envoy_api_field_Listener.tcp_fast_open_queue_length>` option.
```

Comment:
Let's split this into two PRs. One without release notes, just the proto changes, and another to followup post-merge.